### PR TITLE
16543 ineffective menu entry open details view in context menu on small viewports

### PR DESCRIPTION
**Pull request by hschink**
[`* `#16543` Ineffective menu entry "Open Details View" in context menu on small viewports`](https://www.openproject.org/work_packages/16543)

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://fix-16543-ineffective-menu-entry-open-details-view-in-context-m.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

File: app/assets/javascripts/angular/work_packages/work-package-context-menu.js [hunk 1]
```diff
@@ -65,6 +65,9 @@ angular.module('openproject.workPackages')
     $scope.permittedActions = WorkPackageContextMenuHelper.getPermittedActions(getSelectedWorkPackages(), PERMITTED_CONTEXT_MENU_ACTIONS);
   });
 
+  $scope.isDetailsViewLinkVisible = function() {
+    return angular.element('#work-package-context-menu li.open').is(':visible');
```
Comment (myabc) [inline app/assets/javascripts/angular/work_packages/work-package-context-menu.js:69]: @hschink I'm not fond of hard coding HTML ids directly into directives. Can you not a `find` scoped to the current `element`? e.g. `element.find('li.open')`

Comment (hschink) [inline app/assets/javascripts/angular/work_packages/work-package-context-menu.js:69]: @myabc Unfortunately, this is a controller, so `element` does not exist there or do I miss something?

Comment (myabc) [inline app/assets/javascripts/angular/work_packages/work-package-context-menu.js:69]: @hschink you're right. My mistake.

**Comment by myabc** [comment]
@hschink we may also need to adjust resolution here: https://github.com/finnlabs/ci/commit/285836f5e7aac4f0d0cdb95724afc2518cdde8ba

**Comment by mfrister** [comment]
@myabc @hschink Yep, it breaks: http://ci.internal.finnlabs.de/job/config-openproject-org/Db=mysql2,Node=0.10,Ruby=2.1.0,Tests=specs,l=linux/lastCompletedBuild/testReport/

**Comment by mfrister** [comment]
[Adapted the size](https://github.com/finnlabs/ci/commit/88bc64ad80e91059a5c1299f7c4b441b60b9eb94), let's see what happens.

**Pull request merged**